src/pytest_container/container.py

```diff
@@ -280,6 +287,17 @@ def cleanup(self) -> None:
             )
             assert self.host_path
             self._tmpdir.cleanup()
+        else:
+            # Clean up container volume
+            if not os.path.isabs(self.host_path):
+                check_output(
+                    [
+                        container_runtime,
+                        "volume",
+                        "rm", "-f",
+                        self.host_path,
```

Comment:
mypy complaint: List item 4 has incompatible type "Optional[str]"; expected "Union[str, bytes, PathLike[str], PathLike[bytes]]"

Reply:
I kinda don't get that locally. but I added an assert now, maybe that helps

Reply:
you only get this warning when you invoke mypy via `nox -s lint`

Reply:
yeah, not in my setup ?!